src/components/CreateDrinkForm.js

```diff
@@ -0,0 +1,465 @@
+      strIngredient3: data.ingredient3,
+      strIngredient4: data.ingredient4,
+      strIngredient5: data.ingredient5,
+      strMeasure1: data.measure1,
+      strMeasure2: data.measure2,
+      strMeasure3: data.measure3,
+      strMeasure4: data.measure4,
+      strMeasure5: data.measure5,
+    };
+    console.log(newDrink);
+    addNewDrink(newDrink);
+    reset();
+    navigate(-1);
+  };
+
+  return (
+    <FormContainer>
+      <GoBackArrow onClick={() => navigate(-1)}>
+        <img src={goBackArrow} width="35" height="35" alt="go back arrow" />
+        <span className="sr-only">Go back arrow</span>
+      </GoBackArrow>
+      <StyledForm onSubmit={handleSubmit(onSubmit)}>
+        <FormHeadline>Create your own drink!</FormHeadline>
+        <label htmlFor="drinkName">
+          What is the name of your drink?
+          <FormInput
+            type="text"
+            id="drinkName"
+            placeholder="name"
+            autoFocus
+            {...register('drinkName', {
+              required: {
+                value: true,
+                message: 'Your drink needs a name',
+              },
+              minLength: {
+                value: 2,
+                message: 'This name is too short',
+              },
+              maxLength: {
+                value: 30,
+                message: 'This name is too long',
+              },
+            })}
+          />
+          <ErrorMessage>{errors.drinkName?.message}</ErrorMessage>
+        </label>
+
+        <label htmlFor="alcoholic">
+          Alcoholic or non-alcoholic? <br />
+          <FormSelect id="alcoholic" {...register('alcoholic')}>
+            <option value="alcoholic">Alcoholic</option>
+            <option value="non alcoholic">Non-Alcoholic</option>
+          </FormSelect>
+        </label>
+
+        <label htmlFor="category">
+          Category? <br />
+          <FormSelect id="category" {...register('category')}>
+            <option value="Cocktail">Cocktail</option>
+            <option value="Ordinary Drink">Ordinary Drink</option>
+            <option value="Shot">Shot</option>
+            <option value="Coffee / Tea">Coffee / Tea</option>
+            <option value="Soft Drink">Soft Drink</option>
+            <option value="Punch / Party Drink">Punch / Party Drink</option>
+            <option value="Beer">Beer</option>
+          </FormSelect>
+        </label>
+
+        <label htmlFor="glass">
+          Glass? <br />
+          <FormSelect id="glass" {...register('glass')}>
+            <option value="Cocktail glass">Cocktail Glass</option>
+            <option value="Highball glass">Highball Glass</option>
+            <option value="Old-fashioned glass">Old-fashioned Glass</option>
+            <option value="Shot glass">Shot Glass</option>
+            <option value="Beer glass">Beer Glass</option>
+            <option value="Coffee mug">Coffee Mug</option>
+          </FormSelect>
+        </label>
+
+        <IngredientsGrid>
```

Comment:
I think you could make the part of ingredient and measure dynamic. Create  buttons with which it is possible to add and delete fields for ingredients and quantities.

Reply:
Yes, this will come in the next steps so that the form will be more dynamic and with a possibility to upload an own picture. This state is the mvp to have an outcome :-D